### DISCUSSION TITLE
ILLA Builder Future?

### DISCUSSION THREAD

**Discussion by lukastom** [illacloud/illa-builder#3813] [General]
ILLA Builder announcement: "Our cloud service will be discontinued on November 11, 2024. The open-source version remains available." I wanted to build something with this platform but now I don't know. The Discord disappeared which is very bad (how will developers help each other?). Will ILLA builder continue development or you gave up?

**Comment by bili-jing** [comment]
I just wanted to use cloud, will it be gone in the future? And it won't iterate

**Comment by owenlongbo** [comment]
After careful consideration, we have decided to continue our services. Through significant effort, we have resolved our financial challenges and hope to keep serving the open-source community. We deeply appreciate everyone’s support during this time and will strive to make ILLA Builder even better.

Owen, ILLA Cloud CEO

**Reply by lukastom** [reply]
Wow, that is so great! Please, just consider that you need to focus considerably on marketing. Could be even a cheap "guerilla marketing" like posting to low-code Reddit groups. The problem is that ILLA is absolutely unknown to low-code devs. So if there is no marketing, there are no customers, there is no money. Marketing is 50 % of success for you. If you have the best low-code platform in the world but no marketing, all you do is good for nothing.

**Reply by lukastom** [reply]
Will you also open a Discord channel?

**Reply by owenlongbo** [reply]
I’ll be creating a new Discord server within the week. The previous Discord membership expired, so the channel was deleted because it could no longer function.

**Reply by owenlongbo** [reply]
Since we haven’t turned a profit yet, we encountered financial issues over the past few months. But the good news is that we’ve secured new funding, so we’ll keep working hard to build everything up!

**Reply by vprvajrarpv** [reply]
Any news about discord server?

**Comment by owenlongbo** [comment]
I’ll be creating a new Discord server within the week. The previous Discord membership expired, so the channel was deleted because it could no longer function.

**Reply by 67373net** [reply]
做产品不容易，老哥加油👍

**Comment by owenlongbo** [comment]
谢谢哥
